src/Response/TemporaryFileSapiEmitter.hack

```diff
@@ -34,27 +34,33 @@ final class TemporaryFileSapiEmitter implements EmitterInterface {
 
 		$body = $response->getBody();
```

Comment:
you probably want `as File\Handle` here instead of the 4053 fixmes

Reply:
I agree that fooling Hack is bad. Let's hope there aren't any, non File\\Handle instances flowing through this code path? This method was duck typed before.